moveit_ros/planning_interface/move_group_interface/src/wrap_python_move_group.cpp

```diff
@@ -536,11 +536,15 @@ class MoveGroupInterfaceWrapper : protected py_bindings_tools::ROScppInitializer
     }
   }
 
-  Eigen::MatrixXd getJacobianMatrixPython(bp::list& joint_values, bp::list& reference_point_position)
+  Eigen::MatrixXd getJacobianMatrixPython(const bp::list& joint_values,
+                                          const bp::list& reference_point_position = bp::list())
```

Comment:
I suggest to use the default bp::object() here as well (if possible), which would be equivalent to None.
By default, boost::python only exposes the full signature. To also expose other variants (keeping the defaults), you need to perform [overload resolution](https://www.boost.org/doc/libs/1_66_0/libs/python/doc/html/tutorial/tutorial/functions.html#tutorial.functions.overloading).